components/layout_2020/table/layout.rs

```diff
@@ -2862,8 +2862,8 @@ fn get_outer_sizes_for_measurement(
     let min_size = style.min_box_size(writing_mode);
     let max_size = style.max_box_size(writing_mode);
     (
-        outer_size(size.map(|v| get_size_for_axis(v).unwrap_or(Au(0)))),
-        outer_size(min_size.map(|v| get_size_for_axis(v).unwrap_or(Au(0)))),
+        outer_size(size.map(|v| get_size_for_axis(v).unwrap_or(Au::zero()))),
+        outer_size(min_size.map(|v| get_size_for_axis(v).unwrap_or(Au::zero()))),
```

Comment:
Maybe `unwrap_or_else(Au::zero)` to reduce the number of parentheses? But either is fine.

Reply:
I do prefer the `unwrap_or_else` way